docs/requirements.txt

```diff
@@ -1 +1,5 @@
 rst.linker
+mock
```

Comment:
Sounds reasonable. Of course I wanted it to be consistent. I'll try checking whether there's some way to make pip install just extras w/o the package itself. (there's a possibility of bypassing arguments to `setup.py`'s of individual packages, I would play with that)